docs/accessibility.md

```diff
@@ -0,0 +1,89 @@
+Accessibility Guidance
+======================
+
+This document provides guidance and links to useful resources and tools for
+creating web-based content & interfaces which are accessible to the widest
+possible range of users.
+
+It is not intended to be exhaustive but rather to highlight important best
+practices and provide a shared reference which is useful when implementing or
+reviewing UI changes.
+
+## Guidelines
+
+### Mobile
+
+1. **Use media queries to adapt layout to display size** - Use `@media` queries
+   to adjust the layout depending on screen size to provide more usable
+   interfaces on small screens.
+1. **Make interactive controls big enough to tap.** Make sure any UI controls
+   that respond to taps are large enough to tap easily with a finger.
+ * Interactive controls should generally be at least 44x44px in size on
+   touch-screen devices.
+ * Use `@media (pointer: coarse)` to adjust the size of tap-able items
+   on devices with touch screens.
```

Comment:
These two bullet points aren't indented far enough, breaks out of the enumerated list